internal/provider/datasources/workspace_role_test.go

```diff
@@ -20,9 +20,10 @@ func TestAccDatasource_workspace_role_defaults(t *testing.T) {
 
 	resource.ParallelTest(t, resource.TestCase{
 		ProtoV6ProviderFactories: testutils.TestAccProtoV6ProviderFactories,
+		PreCheck:                 func() { testutils.AccTestPreCheck(t) },
 		Steps: []resource.TestStep{
 			{
-				Config: testutils.ProviderConfig + fixtureAccWorkspaceRoleDataSource(owner),
```

Comment:
we actually dont need to re-specify the provider config, there's an implicit, default one that gets loaded. so long as we pass the environment variables, we wont need this one